File: FE/rempms-recruitment-frontend/src/app/model/jobVacancy/JobVacancyRequest.ts
export interface JobVacancyRequest {
  id: number;
  description: string;
  closingDate: string;
  govtJob: boolean;
  walksInInterview: boolean;
  partTime: boolean;
  //posterUrl: string;
  employerId: number;
  jobPositionId: number;
}

File: BE/rempms-recruitment-service/src/main/java/com/pdev/rempms/recruitmentservice/dto/jobVacancy/JobVacancyRequest.java
package com.pdev.rempms.recruitmentservice.dto.jobVacancy;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
public class JobVacancyRequest {
    private Integer id;
    private String description;
    private LocalDate closingDate;
    private boolean govtJob;
    private boolean walksInInterview;
    private boolean partTime;
    private String posterUrl;
    private Integer employerId;
    private Integer jobPositionId;
    private String jobVacancyRefNo;
}
